### PR TITLE
Adds a recharger to Dreamstation's captain's office

### PR DESCRIPTION
**Pull request by ChangelingRain**
this is what I do to issues

actual image

### CODE REVIEW

File: _maps/map_files/DreamStation/dreamstation04.dmm [hunk 1]
```diff
@@ -48832,7 +48832,7 @@
 /area/crew_quarters/captain)
 "bSw" = (
 /obj/structure/table/wood,
-/obj/item/weapon/storage/photo_album,
```
Comment (GunHog) [inline _maps/map_files/DreamStation/dreamstation04.dmm]: Removing the Captain's album? You MONSTER.

Comment (ChangelingRain) [inline _maps/map_files/DreamStation/dreamstation04.dmm]: It happens to be under some donuts.

**Comment by pudl** [comment]
powercreep

**Comment by optimumtact** [comment]
POWAH CREEP

**Pull request merged**